server/pages.js

```diff
@@ -189,10 +187,10 @@ pages.add(
 // ----------
 
 // New collective page - we keep the v2 alias because we shared some of these URLs by email
-pages.add('new-collective-page', '/:slug/v2');
```

Comment:
We don't want that.

Reply:
The comment above says not to remove it because the URLs were shared by email. Do you think it's ok to remove now?

Reply:
It's out of scope. Let's not modify that now.

Reply:
@sbinlondon to be clearer, please revert these changes.